db/system_distributed_keyspace.hh

```diff
@@ -96,11 +96,11 @@ public:
     future<> finish_view_build(sstring ks_name, sstring view_name) const;
     future<> remove_view(sstring ks_name, sstring view_name) const;
 
-    future<> insert_cdc_topology_description(db_clock::time_point streams_ts, const cdc::topology_description&, context);
-    future<std::optional<cdc::topology_description>> read_cdc_topology_description(db_clock::time_point streams_ts, context);
+    future<> insert_cdc_topology_description(db_clock::time_point, const cdc::topology_description&, context);
```

Comment:
I think leaving a generation_id parameter name helps readers understand how to use the function without looking into the implementation so I believe it's worth keeping the parameter name. Just the new one. time_point alone can be anything so it does not help understanding.

Reply:
In the next commit `time_point` is changed to `cdc::generation_id`.